packages/components/organisms/f-checkout/src/services/addressGeocodingApi.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+export default {
+    async getGeoLocation (url, postData, timeout, state) {
+        const authHeader = state.authToken && `Bearer ${state.authToken}`;
+
+        const config = {
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: authHeader
+            },
+            timeout
+        };
+
+        return axios.post(url, postData, config);
```

Comment:
Should this be `async`?

Reply:
Is you're returning, it doesn't needed to be `await`ed :)